src/Codelocks.php

```diff
@@ -67,7 +67,7 @@ public function __construct($key, $accessKey)
 
         // Check that the accessKey is of the right form and at least 10
         // characters
-        if (!preg_match('/^[0-9a-z]{10,}$/', $accessKey)) {
+        if (!preg_match('/^[0-9a-z]{10,}$/', $accessKey) && !preg_match('/^[0-9]{6,}$/', $accessKey) && !is_null($accessKey)) {
```

Comment:
The !is_null() check is not necessary since the regex should already not match a null value.

It might be nice to use a single regex with two possible outcomes i.e.: ^[0-9a-z]{10,}|[0-9]{6,}$ for example: https://regex101.com/r/8ztMeX/1

Regards the additional accessKey format test - are you sure you mean at least 6 characters, or do you mean exactly 6 characters?